index.html

```diff
@@ -107,4 +107,4 @@
             window.addEventListener("load", showPage);
         </script>
 	</body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
I belive this is a result of your computer (Windows) using a diffrent newline character to mine (Mac). Shouldn't actually change anything.